### ISSUE TITLE
On 4.4.1, where queries have random characters appended to the resultant query

### ISSUE THREAD

**Issue by nsamala** [hiddentao/squel#220]
```javascript
.where('l.name != \'parcels\'')
.where(intersectsPointRadiusQuery('geometry', latitude, longitude, radius))
```

When I print out the results of these lines, 
```
WHERE (l.name != 'parcels'll) AND (st_intersects(geometry, ST_Buffer(ST_GeogFromText('SRID=4326; POINT(-75.622365 39.765956)'), 300)::geometry)ss)
```

This seems to happen with all where queries with 4.4.1.

**Comment by krazik** [comment]
others have reported this too

**Comment by hiddentao** [comment]
What version of node are you using? Also, is the problem to do with incorporating the results of your custom function `intersectsPointRadiusQuery` into the query or with _any_ `where()` call?

I just tried the following query in node 0.10, 0.12 and 4 and it worked ok:

```js
var squel = require('squel');
var qry = squel.select().from('table').where('l.name != \'parcels\'');
console.log(qry.toString());
console.log(qry.toParam());
```

output:

```
SELECT * FROM table WHERE (l.name != 'parcels')
{ text: 'SELECT * FROM table WHERE (l.name != \'parcels\')',
  values: [] }
```

**Comment by nsamala** [comment]
The below is the code for that custom function:
```javascript
function intersectsPointQuery(geometryField, latitude, longitude) {
    return "st_intersects({0}, st_setsrid(st_point({1}, {2}), 4326))".format(geometryField, longitude, latitude);
}
```

That issue happened on both Node 5.7.0 (locally) and 4.3.1 (used on AWS). If I revert back to Sequel 4.3.3, I don't have the issue.

**Comment by hiddentao** [comment]
Thanks so much for that. I've finally been able to reproduce it.

**Comment by hiddentao** [comment]
Fixed in 4.4.2. Please try it out and let us know if it works.

**Comment by nsamala** [comment]
Awesome, it looks good on 4.4.2. Thanks!
